### ISSUE TITLE
Serif buttons and labels are off-brand

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#4779]
I'm not sure why the buttons and the date of filing at the top of the new case law UI use a serif font. Let's use our normal buttons for this if we can, or try a couple different designs if our normal buttons don't work for some reason.

**Comment by flooie** [comment]
What about the rest of the caption.

**Comment by flooie** [comment]
![Image](https://github.com/user-attachments/assets/9e378f83-44b0-43a7-8991-1abfd835f168)

**Comment by mlissner** [comment]
Yeah, let's do sanserif for those too. It matches the design, so less change for users, which is good.

**Comment by quevon24** [comment]
> Yeah, let's do sanserif for those too. It matches the design, so less change for users, which is good.

What do you mean to sanserif the buttons and caption? To force the font-family to use a different font? I took a look to css styles and the font is forced into this in the body style: `font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;` and html style also forces sans-serif `font-family: sans-serif;`in old and new case law ui.

i put the old buttons in the new case law ui:

![Image](https://github.com/user-attachments/assets/8884b256-666b-429d-bfa6-61b10a6d8301)

**Comment by mlissner** [comment]
Thanks, you're right, there's not serif fonts anymore, so that's fine. I think we should make the buttons normal though by making them use the `btn-default` class and removing the special CSS that's associated with these buttons.
